btree/btree.c

```diff
@@ -1021,7 +1022,8 @@ struct node_type {
 	void* (*nt_opaque_get)(const struct segaddr *addr);
 
 	/** Captures node data in segment */
-	void (*nt_capture)(struct slot *slot, struct m0_be_tx *tx);
+	void (*nt_capture)(struct nd *node, struct slot *slot,
```

Comment:
Yes it will be present in slot but for a node routine nt_* it makes more sense to take node_descriptor as parameter for getting data within the node_descriptor instead of via the slot pointer. Also in future we might even remove the slot pointer as the node descriptor will contain enough information to indicate what needs to be captured within the node.